engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkStreamingDeploymentManagerProvider.scala

```diff
@@ -30,6 +32,43 @@ class FlinkStreamingDeploymentManagerProvider extends DeploymentManagerProvider
 
   override def typeSpecificInitialData(config: Config): TypeSpecificInitialData = TypeSpecificInitialData(StreamMetaData(Some(1)))
 
+  override def typeSpecificPropertiesConfig(config: Config): Map[String, AdditionalPropertyConfig] = {
+    Map(parallelismConfig, spillStateConfig, asyncInterpretationConfig, checkpointIntervalConfig)
+  }
+
+  private val parallelismConfig: (String, AdditionalPropertyConfig) = "parallelism" ->
+    AdditionalPropertyConfig(
```

Comment:
and `AdditionalPropertyConfig` could be extendend with new field `type` (for now we could support only basic types - String, Boolean, Int, Long, Double), and make FE forms handle those types.